bin/dev

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if ! command -v foreman &> /dev/null
+then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
+foreman start -f Procfile.dev
```

Comment:
why you had to add this file?

Reply:
It was added by the Tailwind gem generator, I thought it was a good addition to run the Procfile.dev easily. But we can remove you think it's not useful.